src/track/replaygain.h

```diff
@@ -98,7 +98,12 @@ class ReplayGain final {
         m_peak = normalizePeak(m_peak);
     }
 
-private:
+    enum ReplayGainUpdateMode {
```

Comment:
enum class?

Reply:
Without registering a corresponding metatype signal with a parameter of this type do not pass thread boundaries, which could be surprising. Those bugs are not caught by at build time and only occur at runtime. The only evidence is a warning message in the log.

Reply:
this is removed anyway